scripts/composer.sh

```diff
@@ -7,6 +7,9 @@ curl -sS https://getcomposer.org/installer | php
 echo "Moving composer to make it globally accessable"
 mv composer.phar /usr/bin/composer
 
+composer config -g notify-on-install false
```

Comment:
It is. But it doesn't have the desired effect anyway.

On Tue, 18 Aug 2015, 5:11 p.m. Will Morgan <notifications@github.com> wrote:

> In scripts/composer.sh
> <https://github.com/BetterBrief/vagrant-skeleton/pull/97#discussion_r37318260>
> :
>
> > @@ -7,6 +7,9 @@ curl -sS https://getcomposer.org/installer | php
> >  echo "Moving composer to make it globally accessable"
> >  mv composer.phar /usr/bin/composer
> >
> > +composer config -g notify-on-install false
>
> This should really be part of travis support, not dev machines
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BetterBrief/vagrant-skeleton/pull/97/files#r37318260>.
>